docs/en/reference/sql/data_types/date_and_time_types.md

```diff
@@ -1,13 +1,13 @@
 # Date and Time Type
 
-OpenMLDB supports date type `DATE` and timestamp `TIMESTAMP`
+OpenMLDB supports date type `DATE` and timestamp `TIMESTAMP`.
 
-Each time type has a valid range of values ​​and a NULL value. The NULL value is used when specifying an invalid value that cannot be represented。
+Each time type has a valid range of values ​​and a NULL value. The NULL value is used when specifying an invalid value that cannot be represented.
 
 | Type      | Size (bytes) |   Scope                                                         | Format            | Use                     |
 | :-------- | :----------- | :----------------------------------------------------------- | :-------------- | :----------------------- |
 | DATE      | 4            | 1900-01-01 ~                                                 | YYYY-MM-DD      | Date Value                   |
-| TIMESTAMP | 8            | End Time is 1970-01-01 00:00:00/2038 **2147483647** Second，Beijing time **2038-1-19 11:14:07**，GMT January 19, 2038 Early Morning 03:14:07 | YYYYMMDD HHMMSS | Mixed Date and Time Value, Timestamp |
+| TIMESTAMP | 8            | From ‘1970-01-01 00:00:01.000000’ UTC to ‘2038-01-19 03:14:07.999999’(2038-01-19 11:14:07.999999 GMT) | online: int64, offline: int64 or 'yyyy-MM-dd'T'HH:mm:ss[.SSS][XXX]' | Mixed Date and Time Value, Timestamp |
```

Comment:
Timezone is default to UTC-8 even just write int64 as unix epoch time. This is a known issue.

Reply:
ok